docs/api-reference.md

```diff
@@ -176,6 +179,39 @@ Creates a new reference object that can be used with [Roact.Ref](#roactref).
 
 ---
 
+### Roact.createContext
+
+!!! danger "Unreleased API"
+	This API is not yet available in a stable Roact release.
+
+	It may be available from a recent pre-release or Roact's master branch.
+
+```
+Roact.createContext(defaultValue: any) -> { Provider, Consumer }
```

Comment:
Is the return type here clear enough? There's more detail in the advanced section, but I'm a bit concerned someone might think the return is an array instead of a table. Not sure how to easily reflect that with the syntax we've been using for type definitions.

Reply:
😭